install_linux.sh

```diff
@@ -68,13 +70,13 @@ else
   dest="${TFLINT_INSTALL_PATH:-/usr/local/bin}/"
   echo "Installing /tmp/tflint to ${dest}..."
   
-  if [[ "$(id -u)" == 0 ]]; then SUDO=""; else
+  if [[ "$(id -u)" == 0 ]] || [[ $TFLINT_INSTALL_NO_ROOT == 1 ]]; then SUDO=""; else
     SUDO="sudo";
   fi
 
   
   $SUDO mkdir -p "$dest"
-  $SUDO install -b -c -v /tmp/tflint "$dest"
+  $SUDO install --backup=none -c -v /tmp/tflint "$dest"
```

Comment:
Perhaps scope creeping a but - but I suggest we change this to not create any backup, to avoid polluting the PATH. What happens now if you run the install script more than once is that you'll end up with a `tflint` and `tflint~` (the backup) in in your install destination.